src/main/java/org/jenkinsci/test/acceptance/po/FormValidation.java

```diff
@@ -88,14 +89,16 @@ public static FormValidation await(Control control, boolean silent) {
             element.sendKeys(Keys.TAB);
 
             // Wait for validation area to stop being <div></div>
-            validationArea = control.waitFor().until(() -> {
-                // path to validation area is the parents sibling with class `validation-error-area`
-                String xpath = silent
-                        ? "../../div[contains(@class,'validation-error-area')]"
-                        : "../../div[contains(@class,'validation-error-area--visible')]";
-
-                return element.findElement(by.xpath(xpath));
-            });
+            validationArea = control.waitFor()
+                    .ignoring(NoSuchElementException.class)
```

Comment:
has to ignore the NoSuchElement or this would only ever run once...
FOrmValidationois most generally asynchronous :)